guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
+
+Examples of UI or local state include:
+
+* selection 
+* focus 
+* animation state 
+* occlusion state
+* information specific to the local display of a single component 
```

Comment:
Let's target this section more directly on issues we're seeing in apps. Are people storing a lot of occlusion state on ember-data models? If not, let's not mention it.